paper-tooltip.js

```diff
@@ -40,23 +48,14 @@ Custom property | Description | Default
 `--paper-tooltip` | Mixin applied to the tooltip | `{}`
 `--paper-tooltip-delay-in` | Delay before tooltip starts to fade in | `500`
 `--paper-tooltip-delay-out` | Delay before tooltip starts to fade out | `0`
-`--paper-tooltip-duration-in` | Duration of animation when showing tooltip | `500`
+`--paper-tooltip-duration-in` | Duration of animation when showing tooltip |
+`500`
```

Comment:
This table was broken by the formatter. ([link to fix](https://github.com/PolymerElements/iron-fit-behavior/pull/93#discussion_r199271183))

Reply:
used shorter words 🙄